server/routes/treeRouter.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const treeRouter = express.Router();
+const helper = require('./utils');
+const db = require('../database');
+
+treeRouter.get(
+  '/potential_matches',
+  helper.handlerWrapper(async (req, res) => {
+    const id = req.query.capture_id;
+    //maximum distance in meters between possible matches and tree in query
+    const distance = 6;
+    const query =
+      'SELECT id, image_url as "imageUrl", lat, lon, species_id as "speciesId", morphology, age, status, time_created as "createdAt", time_updated as "updatedAt" FROM trees WHERE ST_DWithin(estimated_geometric_location::geography, (SELECT estimated_geometric_location FROM trees WHERE id=$1)::geography, $2)';
```

Comment:
This query would be correct for the current legacy database, but this microservice comes with a new schema.  The query now becomes


'SELECT id, image_url as "imageUrl", lat, lon, species_id as "speciesId", morphology, age, status, time_created as "createdAt", time_updated as "updatedAt" FROM tree WHERE ST_DWithin(estimated_geometric_location::geography, (SELECT estimated_geometric_location FROM capture WHERE id=$1)::geography, $2)';

Note the change of the first reference to 'trees' to 'tree', and the second from 'trees' to 'capture'

Reply:
Thanks! I made the changes to reflect the different schema and the new `estimated_geographic_location` column as well.